### PR TITLE
[Feature] Personal Information Form Backend Validation 

### PR DESCRIPTION
**Pull request by j-bunk**
## Notion ticket link

[Personal Information Form Backend Validation](https://www.notion.so/uwblueprintexecs/Personal-Information-Form-Backend-Validation-f5f911bd3d5e4657b34fb8227e12740e)

## Implementation description
* Closely follows the implementation of https://github.com/uwblueprint/richmond-centre-for-disability/pull/204

Example of how the backend form validation error is displayed (the error was manually triggered):

### Permit-holder page
image

### Request page (for new request)
image

### Request page (renewal request)
image

## Notes
* 

## Checklist
- [x] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: tools/admin/requests/permit-holder-information.ts [hunk 1]
```diff
@@ -158,13 +159,18 @@ export const UPDATE_NEW_APPLICATION_PERMIT_HOLDER_INFORMATION = gql`
...
       ok
+      error
     }
   }
 `;
 
 export type UpdateNewApplicationPermitHolderInformationRequest =
   MutationUpdateNewApplicationGeneralInformationArgs;
 
+export type UpdateNewPermitHolderInformationResponse = {
```
Comment (OustanDing) [inline tools/admin/requests/permit-holder-information.ts:170]:
```suggestion
export type UpdateNewApplicationPermitHolderInformationResponse = {
```

File: lib/applicants/resolvers.ts [hunk 2]
```diff
@@ -234,8 +235,20 @@ export const updateApplicantGeneralInformation: Resolver<
   MutationUpdateApplicantGeneralInformationArgs,
   UpdateApplicantGeneralInformationResult
 > = async (_parent, args, { prisma }) => {
-  // TODO: Validation
   const { input } = args;
+
+  try {
+    await permitHolderInformationSchema.validate(input);
+    throw Error;
```
Comment (OustanDing) [inline lib/applicants/resolvers.ts:242]: Remove

File: lib/applicants/validation.ts [hunk 3]
```diff
@@ -44,6 +44,14 @@ export const editRequestPermitHolderInformationSchema = object({
   permitHolder: requestPermitHolderInformationSchema,
 });
 
+/**
+ * Validation schema for request-side permit holder information mutation
+ */
+export const requestPermitHolderInformationMutationSchema =
+  requestPermitHolderInformationSchema.shape({
+    id: number().positive('Application not found').required('Application not found'),
```
Comment (OustanDing) [inline lib/applicants/validation.ts:52]: Fix error messages based on previous comment

**Review by OustanDing — CHANGES_REQUESTED** [review]

File: tools/admin/requests/permit-holder-information.ts (hunk 1 above)
Comment (j-bunk) [reply tools/admin/requests/permit-holder-information.ts]: Addressed!

File: lib/applicants/resolvers.ts (hunk 2 above)
Comment (j-bunk) [reply lib/applicants/resolvers.ts]: Addressed!

File: lib/applicants/validation.ts (hunk 3 above)
Comment (j-bunk) [reply lib/applicants/validation.ts]: Addressed!

**Comment by j-bunk** [comment]
@OustanDing Unrelated to my changes but I noticed you can select `Other` as a gender but there isn't a way to specify what that gender is

**Comment by OustanDing** [comment]
@j-bunk Thanks for bringing that up, noted

**Review by OustanDing — APPROVED** [review]

**Pull request merged**